bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -82,18 +95,19 @@
  */
 public class EntryLogger {
     private static final Logger LOG = LoggerFactory.getLogger(EntryLogger.class);
+    private static final Long INVALID_LEDGERID = new Long(-1);
+    // log file suffix
+    private static final String LOG_FILE_SUFFIX = ".log";
 
-    private static class BufferedLogChannel extends BufferedChannel {
+    class BufferedLogChannel extends BufferedChannel {
```

Comment:
you change this to inner class, because of line 132. however I am not sure we should add `isLedgerDirFull` should be part of `BufferedLogChannel`. `isSomethingFull` should be part of ledger dirs manager, not part of an entry log channel.